cucumber/authenticators_oidc/features/support/scenario_context.rb

```diff
@@ -0,0 +1,25 @@
+module Utilities
+  class ScenarioContext
```

Comment:
I really like handling shared between steps like this. Keeps the relevant data a lot more visible than just sticking values in random instance variables.